docs/js-core-1/week-1/learning-objective.md

```diff
@@ -0,0 +1,30 @@
+---
+id: learning-objectives
+title: Learning Objectives
+sidebar_label: Learning Objectives
+---
+
+_The learning objectives under a given topic may be achieved over the multiple weeks of the module, rather than in the first week the topic is introduced._
+
+## PD (team-working and communication)
+- [ ] Demonstrate reliability
+- [ ] Meet commitments
+- [ ] Motivate others 
+- [ ] Recognise and relate to different perspectives 
+- [ ] Recognise when conflict may arise 
+- [ ] Recognise personal strengths and weaknesses in group setting
+- [ ] Resolve conflicts/disagreements
+- [ ] Demonstrate willingness and ability to achieve a goal in cooperation with others through retrospective reflection 
+- [ ] Submit project with the group with balanced distribution of tasks
+- [ ] Recognise and understand different perspectives
+- [ ] Mitigate conflict in a productive manner 
+- [ ] Articulate learnings through recognition and conflict resolution in homework
```

Comment:
I think some of these objectives could also do with being checked with a grammar tool.  I don't know what this sentence means at all 

" - [ ] Articulate learnings through recognition and conflict resolution in homework"